openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/GlossaryTermRepository.java

```diff
@@ -99,6 +99,12 @@ public class GlossaryTermRepository extends EntityRepository<GlossaryTerm> {
   private static final String UPDATE_FIELDS = "references,relatedTerms,synonyms";
   private static final String PATCH_FIELDS = "references,relatedTerms,synonyms";
 
+  /*
```

Comment:
we cannot do it this way , this is not the right approach

Reply:
removed that change, currently  supports for changing the parent once in session